settings/settings-vllm-pgvector.yaml

```diff
@@ -4,7 +4,6 @@ server:
 
 llm:
   mode: openailike
-  tokenizer: ${VLLM_TOKENIZER:lmsys/vicuna-7b-v1.5}
```

Comment:
let's also get rid of this one https://github.com/neuro-inc/private-gpt/blob/apolo/settings/settings-app.yaml#L7